### DISCUSSION TITLE
Styling DatePicker

### DISCUSSION THREAD

**Discussion by jsb1138** [mantinedev/mantine#4405] [Q&A]
Hi, I've got a couple questions about styling. I'm having a hard time doing everything I want and I'm using getDayProps for most of what I've already done, but the logic is piling up and I've got quite a lot of lines just to style the days the way I want, with some things outstanding:

1. Is there a simple way to change the text color of the visible outside dates? (The few dates from last month and next month which are visible this month)
2. How on earth do you style the prospective range of dates that appear after you've selected one date and move your mouse over other dates?

Thanks in advance!

**Comment by rtivital** [comment]
1. Yes, days have `data-outside` attribute which is set when day is outside of the current month, you can use it to apply styles
2. On earth, you can use `data-in-range`, `data-first-in-range` and `data-last-in-range` attributes (may or may not work on the Moon)

**Reply by jsb1138** [reply]
Thanks for the fast reply! Can confirm it also works on the moon.

**Reply by rtivital** [reply]
Great news, now only an expedition on Uranus left to verify that the library works well in the entire solar system

**Comment by AshutoshDash1999** [comment]
@jsb1138 How have you styled a selected date using getDayProps?

**Reply by jsb1138** [reply]
I had used the date-fns package and did checks to see if the chosen date matched certain criteria, but it's all made much easier when you use the in-built attributes. 

`  [data-today]:hover {
    border: 1px solid YOUR-COLOR;
  }`

For example, that will add a hover state border to today's date.

If I recall, the attributes are in the V5 docs...

**Reply by AshutoshDash1999** [reply]
But I am using V6, I guess they are not working anymore. How can I style the selected date using JS?
